ledger-api/grpc-definitions/com/daml/ledger/api/v1/command_completion_service.proto

```diff
@@ -60,6 +60,10 @@ message CompletionStreamRequest {
   // This offset is exclusive: the response will only contain commands whose offset is strictly greater than this.
   // Optional, if not set the ledger uses the current ledger end offset instead.
   LedgerOffset offset = 4;
+
+  // Must be a valid domain ID
```

Comment:
I thought the plan was to make domain ids mandatory. What do I get if I don't specify it here?

Reply:
outdated, this param is not existent anymore (removed in transitioning the api to participant offsets)